### DISCUSSION TITLE
How to initialize existing positions at the start of the strategy

### DISCUSSION THREAD

**Discussion by Xuan-Zheng-Xie** [nautechsystems/nautilus_trader#1780] [Q&A] [answered]
The reconciliation process seems to require that there are no positions in the LiveExecutionEngine.

**Comment by cjdsellers** [comment, marked as answer]
Hi @Xuan-Zheng-Xie

There have been numerous improvements to the reconciliation procedure more recently.
There is no such limitation, and you can read more in the docs [here](https://nautilustrader.io/docs/latest/concepts/execution#execution-reconciliation).

I hope that helps!
